index.html

```diff
@@ -854,18 +854,14 @@ <h2 id="docconformance">
               <a>No corresponding role</a>
             </td>
             <td>
-              <p>
-                Roles:
-                <a href="#index-aria-application">`application`</a>,
-                <a href="#index-aria-document">`document`</a>,
-                <a href="#index-aria-img">`img`</a>,
-                <a href="#index-aria-presentation">`presentation`</a>
-                or <a href="#index-aria-none">`none`</a>.
```

Comment:
I don't think this is a good change. Role=none on iframes serves a very important function for VoiceOver. It's a way to have iframe content treated as part of the same page, rather than a completely isolated context.

Reply:
@WilcoFiers did you see the discussion at https://github.com/w3c/html-aria/issues/368 ?

Reply:
Yes I did.

Reply:
these work with webkit, for now...

ignored with firefox (because iframes are focusable).

even if they suppress the iframe discoverability in the primary document, once navigating into the contents i'm finding that 'frame' is still announced.    will post more results later, but really the only major benefit here is obscuring iframes from macOS webkit / VO users.

Reply:
I seem to recall this being advertised as a best practice when ARIA first came out. There was a lot of misunderstanding of what role=presentation meant on an iframe, but ultimately, it does have an intended purpose - to communicate to the browser and AT that the content of the frame should be treated as part of the parent page and not as a sub-page. The fact that some browsers and ATs don't support this doesn't change the fact that the spec intended for the browsers to support this scenario. I think ARIA and the browser vendors need to determine who's going to budge, but as it stands now, this is perfectly valid and meaningful to put in your code.

Reply:
See https://github.com/w3c/html-aria/pull/370#issuecomment-976111535. I'm not saying the ARIA spec should be changed. But this spec is primarily targeted at conformance checkers. It seems pretty bad to have conformance checkers telling authors something is conformant when it doesn't work across multiple browsers. Yes, we should figure out a path forward on that, but until then, isn't it better to avoid user pain by advising authors to avoid doing things which don't work across browsers?

Reply:
agree with @jcsteh.

i'm sorry i haven't gotten around to compiling all my test data yet.  it's coming... just have to work through other things first.

Reply:
That depends on the intent of your checker. If checkers only made decisions based on broad browser support, every checker would have flagged any use of ARIA as a failure when ARIA was first released since basically only one browser supported it at the time. Some checkers focus more on technical specifications whereas others look more at best practices - there's a spectrum. This issue feels like it falls somewhat inbetween, and this PR feels like it's trying to deprecate these scenarios and codify it into technical specification based on what's in the wild rather than on whether or not it makes sense to do so.

The unfortunately bit with this particular case is that this was heavily advertised in the early days as an intended pattern. Wilco's note that there exists an implementation that meets that pattern reinforces that. So, it's hard to say it can't be done when it has obviously been done.

So far, I haven't seen a discussion as to whether this pattern makes sense or not - just some notes that it's not easy to support it.